dapr/proto/scheduler/scheduler.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+
+package dapr.proto.scheduler.v1;
+
+import "google/protobuf/empty.proto";
+import "dapr/proto/runtime/v1/dapr.proto";
+
+service Scheduler {
+	// ConnectHost is used by the daprd sidecar to connect to the scheduler service.
+	// The sidecar will send a stream of RegisterRequest, UnregisterRequest and blank 
+	// health-check messages.
+	// In return, scheduler service will send a stream of blank ConnectServerStream messages,
+	// which will also act as health check messages.
+	rpc ConnectHost(ConnectClientStream) returns (ConnectServerStream) {}
```

Comment:
I'd like to remove the streaming bits for now, since its not in the proposal. The register api vs streaming can be decided once more of the logic is implemented and we get to this point. I'd like to stick to the proposal outline as much as possible until we find gaps or see a reason to change the proposal